updater/spec/github_api/dependency_submission_spec.rb

```diff
@@ -62,25 +68,25 @@
       {
         context: "with a typical RubyGems project in directory root",
         directory: "/",
-        expected_correlator: "dependabot-bundler-Gemfile.lock"
+        expected_correlator: "dependabot-bundler-"
```

Comment:
```suggestion
        expected_correlator: "dependabot-bundler"
```

We should probably trim this trailing `-` for root directories, we could use a placeholder token of `root`, but I strongly suspect that'd create annoying edge cases if someone actually had a repository with `/root` in it...